### DISCUSSION TITLE
Github Student Developer Pack Application not getting submitted

### DISCUSSION THREAD

**Discussion by AdityaKulkarni** [community/community#65798] [GitHub Education]
### Select Topic Area

Bug

### Body

I am applying for the GitHub student developer pack. In the application, it asks for a proof for which I am uploading my admit letter screenshot. But after the upload, when I click process application, it doesn't do anything.

I am attaching screenshots of the process. The first image is after I have uploaded the photo proof, second image is after I click the "Process my application" button, and the last image is after the process is complete, it brings me back to the same page.

 Any help is appreciated. Thank you.
![1](https://github.com/community/community/assets/25547842/3dd56a21-5e78-4fd0-a5cb-72c6b93ece89)
![2](https://github.com/community/community/assets/25547842/db73f163-c6c1-4899-b1d2-06319a987900)
![3](https://github.com/community/community/assets/25547842/42771caa-b354-4621-8f1e-4e8b67ebfd32)

**Comment by Amirkhsh92** [comment]
I have the same issue.
Could you solve it?

**Comment by Angel030331** [comment]
Try to take a photo instead of uploading it. I faced the same issue when I was trying to upload the documents but I can proceed with I took a photo.

**Comment by karloskarl** [comment]
I am having the exact same issue
